frontend/src/static/js/v_summary.js

```diff
@@ -236,7 +236,17 @@ window.vSummary = {
 
       return totalBars;
     },
-
+    getFileFormatsTouchedInRepo(repo) {
```

Comment:
Is it implied or necessary to name it with `Touched`?

Usually less is better, no need to keep a mental note or define its meaning.